tests/testdata/prob_dists.mal

```diff
@@ -0,0 +1,16 @@
+// vim:ft=mal
+#id: "test"
+#version: "0.0.0"
+
+category Test{
+  asset Dummy {
+    | expo [Exponential(0.1)]
+    | berni [Bernoulli(0.5)]
+    | composite [Exponential(0.1) * Bernoulli(0.5)]
+    | compositeComplex [Bernoulli(0.5) * (Binomial(10, 0.1) + Exponential(0.2) - Gamma(0.1, 0.2) * LogNormal(5, 0.5) / Pareto(7, 0.25) ^ TruncatedNormal(6, 0.3) + Uniform(0.1, 0.9))]
```

Comment:
@nkakouros just as a heads up, we don't need it right now since it will not be used any time soon, but the compiler does not handle the `/` correctly. The expression simply terminates when it encounters it.